api/data.go

```diff
@@ -105,14 +106,19 @@ func (api *API) GetCacheTime(ctx context.Context, w http.ResponseWriter, req *ht
 
 	cacheTime, err := api.dataStore.GetCacheTime(ctx, id)
 	if err != nil {
-		log.Error(ctx, "error retrieving cache time from datastore", err)
-		http.Error(w, "Internal Server Error", http.StatusInternalServerError)
+		if err == errs.ErrCachetimeNotFound {
```

Comment:
This is apparently no longer good practice. See https://stackoverflow.com/a/57613539 for an explanation and an alternative solution.